xcode/Subconscious/Shared/Components/Detail/DetailMetaSheet.swift

```diff
@@ -0,0 +1,307 @@
+//
```

Comment:
This new subcomponent exposes "meta" controls for notes, such as:

- Change audience
- Change link
- Delete

And in future will be a good place to put note history, IPFS hash, sphere ID, etc.